src/profile-logic/import/chrome.js

```diff
@@ -390,172 +419,181 @@ async function processTracingEvents(
 
   const threadInfoByPidAndTid = new Map();
   const threadInfoByThread = new Map();
-  for (const chunkOrCpuProfileEvent of profileEvents) {
+  for (const profileEvent of profileEvents) {
     // The thread info is all of the data that makes it possible to process an
     // individual thread.
     const threadInfo = getThreadInfo(
       threadInfoByPidAndTid,
       threadInfoByThread,
       eventsByName,
       profile,
-      chunkOrCpuProfileEvent
+      profileEvent
     );
     const {
       thread,
       funcKeyToFuncId,
       nodeIdToStackId,
       originToResourceIndex,
     } = threadInfo;
-    const { cpuProfile } = chunkOrCpuProfileEvent.args.data;
-    const { nodes, samples } = cpuProfile;
-    const timeDeltas = getTimeDeltas(chunkOrCpuProfileEvent);
-    if (!timeDeltas) {
-      continue;
-    }
 
-    if (cpuProfile.startTime !== undefined) {
-      threadInfo.lastSeenTime = (cpuProfile.startTime: any) / 1000;
+    let profileChunks = [];
```

Comment:
Disclosure: I mostly skimmed this commit's changes, as they are low-risk since we don't have a lot of users yet, and I trust that they are probably mostly correct :)